stories/Atom/Navigation/Scroll-bar/ScrollBar.stories.mdx

```diff
@@ -0,0 +1,37 @@
+import { Meta, Story } from '@storybook/addon-docs';
+import { ScrollBar } from './ScrollBar';
+
+<Meta title="Atoms/Navigation/ScrollBar"/>
```

Comment:
@harishkumar-srijan Please correct the placement of the component as per the tree structure